CHANGELOG.md

```diff
@@ -76,6 +76,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed an issue with `_auto_collect_arguments` collecting local variables that are not constructor arguments and not working for signatures that have the instance not named `self` ([#2048](https://github.com/PyTorchLightning/pytorch-lightning/pull/2048))
 - Fixed mistake in parameters' grad norm tracking ([#2012](https://github.com/PyTorchLightning/pytorch-lightning/pull/2012))
 - Fixed CPU and hanging GPU crash ([#2118](https://github.com/PyTorchLightning/pytorch-lightning/pull/2118))
+- Fix num batches in case of multiple dataloaders and percent_check ([#1920](https://github.com/PyTorchLightning/pytorch-lightning/pull/1920))
```

Comment:
```suggestion
- Fixed number batches in case of multiple dataloaders and `percent_check` ([#1920](https://github.com/PyTorchLightning/pytorch-lightning/pull/1920))
```